Puppetfile.tracking

```diff
@@ -1,59 +1,62 @@
-# refs recorded for SIMP release 6.0.0
-
 moduledir 'src'
 
 mod 'simp-doc',
   :git => 'https://github.com/simp/simp-doc',
-  :tag => '6.2.0-0'
+  :branch => 'master'
 
 moduledir 'src/assets'
 
 mod 'rubygem-rubygem_simp_cli',
   :git => 'https://github.com/simp/rubygem-simp-cli',
-  :ref => '4.2.0'
+  :branch => 'master'
+
+# FIXME Doesn't have RPM packaging yet
+#mod 'rubygem-rubygem_simp_processgraph',
+#  :git => 'https://github.com/simp/rubygem-simp-processgraph',
+#  :ref => 'master'
 
 mod 'simp-rsync_data',
   :git => 'https://github.com/simp/simp-rsync-skeleton',
-  :ref => '6.2.1'
+  :branch => 'master'
 
 mod 'simp-adapter',
- :git => 'https://github.com/simp/simp-adapter',
- :branch => 'master'
+  :git => 'https://github.com/simp/simp-adapter',
+  :branch => 'master'
 
 mod 'simp-environment',
   :git => 'https://github.com/simp/simp-environment-skeleton',
-  :ref => '6.2.10'
+  :branch => 'master'
 
 mod 'simp-gpgkeys',
   :git => 'https://github.com/simp/simp-gpgkeys',
-  :tag => '3.0.3'
+  :branch => 'master'
 
 mod 'simp-utils',
   :git => 'https://github.com/simp/simp-utils',
-  :tag => '6.1.1'
+  :branch => 'master'
 
 moduledir 'src/puppet/modules'
 
 mod 'camptocamp-kmod',
   :git => 'https://github.com/simp/puppet-kmod.git',
   :tag => '2.2.0'
 
-# We are maintaining a simp-master branch of this module
-# because camptocamp has moved to Hiera 5 and Puppet >=
-# 4.10.10.  We need to release our version of 1.1.1
-# which is on simp-master.
-mod 'simp-systemd',
+# Revert to camptocamp/systemd for Puppet 5
+# TODO Verify upgrade works switching from simp-systemd to camptocamp-systemd
+mod 'camptocamp-systemd',
   :git => 'https://github.com/simp/puppet-systemd.git',
-  :tag => 'simp-1.1.1'
+  :tag => '2.1.0'
```

Comment:
Due to the need for RPM triggers, and the complexity of getting this working in the Oct time frame, I think that we should just update our fork until we can properly fix the module deployment strategy.